Tests/Service/ServiceFacadeTest.php

```diff
@@ -25,7 +25,7 @@ protected function setUp()
         $this->service = ServiceFacade::getInstance($this->logger);
         $this->formats = array('json', 'object', 'xml');
         $this->config = array(
-            'url' => 'http://navitia2-ws.ctp.dev.canaltp.fr',
+            'url' => 'http://navitia2-ws.ctp.internal.canaltp.fr',
```

Comment:
dev was dead the day I made this PR so I had no other choice to run the tests...